pageserver/src/tenant.rs

```diff
@@ -118,6 +118,10 @@ pub struct Tenant {
     // Global pageserver config parameters
     pub conf: &'static PageServerConf,
 
+    /// The creation timestamp, used to measure activation delay, see:
+    /// <https://github.com/neondatabase/neon/issues/4025>
+    created_at: Instant,
```

Comment:
```suggestion
    /// The timestamp when the Tenant object was created.
    object_created_at: Instant,
```

Reply:
But we don't have objects, we have values. I don't see the point of PL lingo here.

Reply:
`value_created_at` then.

The important part is that it's not called plain "created", because that can be confused with the act of creating the durable tenant on-disk state.

Reply:
Hmm... ok, I see your point. I think `loading_started_at` is what it is used as.

Reply:
`loading_started_at` in 3efeccb2274c753f7fa1f94501048616b0be38a6